static/images/favicon/safari-pinned-tab.svg

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 20010904//EN"
+ "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
+<svg version="1.0" xmlns="http://www.w3.org/2000/svg"
+ width="260.000000pt" height="260.000000pt" viewBox="0 0 260.000000 260.000000"
+ preserveAspectRatio="xMidYMid meet">
+<metadata>
+Created by potrace 1.14, written by Peter Selinger 2001-2017
+</metadata>
+<g transform="translate(0.000000,260.000000) scale(0.100000,-0.100000)"
+fill="#000000" stroke="none">
+<path d="M1150 2593 c-14 -2 -61 -11 -105 -20 -536 -105 -953 -545 -1032
+-1088 -10 -66 -10 -284 -1 -360 4 -33 15 -87 23 -120 9 -33 17 -67 19 -75 1
+-8 6 -24 10 -35 4 -11 17 -45 29 -75 39 -104 136 -270 199 -341 13 -14 37 -41
+53 -60 85 -94 234 -209 355 -272 60 -31 186 -83 222 -92 191 -46 246 -53 398
+-50 164 3 247 16 385 61 106 35 263 110 315 151 8 7 39 29 67 51 261 192 443
+498 498 835 13 81 14 330 1 392 -4 17 -11 55 -17 85 -41 208 -166 442 -324
+612 -195 207 -456 345 -748 395 -39 6 -313 11 -347 6z"/>
+</g>
+</svg>
```

Comment:
Is it just me or this SVG is broken?
<img width="1753" alt="Screenshot 2022-01-18 at 12 50 52" src="https://user-images.githubusercontent.com/1009343/149971167-3e0fc72f-ff36-4a7e-854a-a94971a06345.png">

Reply:
You're right it's broken

Reply:
But I've a doubt, it's because it's safari pinned tabs, maybe there is a trick where safari pinned tab apply his own colors to the SVG

Reply:
Or @okom3pom didn't play with the contrast while building the favicon, you've a little slider that you can move in order to make the logo visible